renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":disableRateLimiting",
+    ":maintainLockFilesWeekly",
+    ":semanticCommits"
```

Comment:
Enable semantic commit. It may not be needed, if so you can disable it with: `":semanticCommitsDisabled"`